### ISSUE TITLE
N+1 issue on dashboard counter

### ISSUE THREAD

**Issue by webdistortion** [monicahq/monica#135]
Some of the dashboard code for counting Kids etc. is iterating through each contact. from looking at a laravel debugger in the browser ive a feeling this looks like it is creating a sql statement for each contact rather than a single sum operation. Efficiency could be improved here.

**Comment by djaiss** [comment]
@webdistortion definitely. As with most of the codebase, it's far from being optimized right now.

**Comment by Lovatt** [comment]
I'll submit a pull for this tonight, i've been working on another n+1's elsewhere in the app #138
